src/components/learner-credit-management/utils.jsx

```diff
@@ -0,0 +1,6 @@
+import dayjs from 'dayjs';
+
+// eslint-disable-next-line import/prefer-default-export
+export function formatDate(date) {
```

Comment:
Might recommend moving this function to the existing `utils.js` function for the LCM feature (`/src/components/learner-credit-management/data/utils.js`) as opposed to creating another utils file.